### DISCUSSION TITLE
Enhancement:

### DISCUSSION THREAD

**Discussion by glitch090** [FunkinCrew/Funkin#1642] [Ideas]
## What is your suggestion, and why should it be implemented?

I think would be great to put a button to return to main menu when you are in story mode,options and freeplay because i don't see a button like this on friday night funkin

**Comment by UnknownSkid** [comment]
what?

**Comment by DuckBergen** [comment]
> #### Please check for duplicates or similar issues before creating this issue.
> ## What is your suggestion, and why should it be implemented?
> I think would be great to put a button to return to main menu when you are in story mode,options and freeplay because i don't see a button like this on friday night funkin

you can use the esc key to return to main menu
